package.json

```diff
@@ -37,18 +37,23 @@
   "dependencies": {
     "acorn": "^8.14.0",
     "husky": "9.1.7",
-    "lint-staged": "15.3.0",
-    "turbo": "2.3.3"
+    "lint-staged": "15.4.3",
+    "turbo": "2.4.4"
   },
   "devDependencies": {
-    "@eslint/js": "~9.17.0",
-    "@types/eslint__js": "8.42.3",
-    "@types/node": "22.10.3",
+    "@eslint/js": "~9.21.0",
+    "@types/node": "22.13.8",
+    "@types/react": "^19.0.10",
+    "@types/react-dom": "^19.0.4",
```

Comment:
why needed on root of monorepo ?

Reply:
Because overrides only work on root of monorepos.

Reply:
ohhh sad. Have you the list of dep that we need to overwrite ?